web/src/app/core/services/repository.service.ts

```diff
@@ -74,6 +74,19 @@ export class RepositoryService {
     return of(new RepositoryModel(callable({ repositoryUid: repo.uid, token: this.authService.profile.oauth.githubToken })));
   }
 
+  /**
+   * Call cloud function create webhook manually
+   * @param repo repository
+   */
+  public createGitWebhooks(repos: IRepository[]): Observable<RepositoryModel> {
+    const callable: any = this.fns.httpsCallable('createGitWebhooks');
+
+    return of(new RepositoryModel(callable({
+      repositoryUids: repos.map((repo: IRepository) => repo.uid),
+      token: this.authService.profile.oauth.githubToken,
```

Comment:
I dont think we need to pass the GitHub token anymore

Reply:
that is missed . will remove it .